src/bundle/Controller/ContentViewController.php

```diff
@@ -52,6 +52,8 @@ public function __construct(
 
     public function locationViewAction(ContentView $view)
     {
+        $view->setCacheEnabled(false);
```

Comment:
should probably have had a comment here to explain why we disable it.

Reply:
@andrerom A bit late but here it is: https://github.com/ezsystems/ezplatform-admin-ui/commit/aab8de44534b537f31ac8409b517e36ddc063656 (already on master)